### ISSUE TITLE
Multiple Attributes Seperated By Comma Not Working

### ISSUE THREAD

**Issue by whinee** [casey/just#1610]
Hello, here is the relevant recipe:

```just
# Set up development environment
[linux, macos, unix]
bootstrap:
    #!/usr/bin/env bash
    rm -rf poetry.lock
    if test ! -e pyenv; then
        {{ system_python }} -m venv pyenv
        {{pyenv_activate_cmd}}
    fi
    {{ python }} -m pip install --upgrade pip poetry
    {{ python }} -m poetry install --with dev
    {{ python }} -m pip cache purge
```

It yields the following error:

```text
error: Expected ']', but found ','
   |
58 | [linux, macos, unix]
   |       ^
error: Expected ']', but found ','
   |
58 | [linux, macos, unix]
   |       ^
```

According to the documentation, it should work:

> Or separated by commas on a single line:
> 
> ```just
> [no-cd, private]
> foo:
>    echo "foo"

But clearly, it does not. Changing it to the following makes it work:

```just
# Set up development environment
[linux]
[macos]
[unix]
bootstrap:
    #!/usr/bin/env bash
    rm -rf poetry.lock
    if test ! -e pyenv; then
        {{ system_python }} -m venv pyenv
        {{pyenv_activate_cmd}}
    fi
    {{ python }} -m pip install --upgrade pip poetry
    {{ python }} -m poetry install --with dev
    {{ python }} -m pip cache purge
```

**Comment by neunenak** [comment]
The commit that added support for this syntax (https://github.com/casey/just/commit/687831816db94c6d3364b4c3ca8d974f83741dd8) was merged a couple of days after the last `just` release, 1.13.0.  The next time a new version of `just` is released this change should get picked up, or alternatively you can compile the latest `master` branch from source yourself.

The public documentation confusingly is built from the state of the documentation as of the latest `master` branch, not the latest release. So it's possible for the documentation to get out of sync with what the actual state of the latest-officially-released code is.

**Comment by whinee** [comment]
I can't check it right now as I am about to sleep. But, is there like documentation for every single version release out there? If not, isn't that a bad idea?

**Comment by casey** [comment]
I just published 1.14.0, so this should be working as of the latest published version of Just. Sorry for the delay!

**Comment by whinee** [comment]
Thank you!
